includes/discount-functions.php

```diff
@@ -33,7 +33,8 @@ function edd_get_discounts( $args = array() ) {
 	$args = wp_parse_args( $args, $defaults );
 
 	$discounts_hash = md5( json_encode( $args ) );
-	if ( ! $discounts = wp_cache_get( $discounts_hash, 'edd-discounts' ) ) {
+	$discounts = $discounts = wp_cache_get( $discounts_hash, 'edd-discounts' );
```

Comment:
Needs changing to `$discounts = wp_cache_get( $discounts_hash, 'edd-discounts' );`

Reply:
Oops! Fixed that.